tests/exception-policy-midstream-05/test.yaml

```diff
@@ -0,0 +1,18 @@
+args:
+- --simulate-ips
+- --set stream.midstream-policy=bypass
+
+checks:
+  - filter:
+      count: 0
+      match:
+        event_type: alert
+  - filter:
+      count: 1
+      match:
+        event_type: flow
+        flow.action: pass
```

Comment:
does the flow have something to indicate it was bypassed?

Reply:
yes, sorry, I'll add a check for the flow state.